deployer/v2/config/manifests/bases/redhat-marketplace-operator.clusterserviceversion.yaml

```diff
@@ -105,7 +105,16 @@ spec:
     Hat Marketplace products, you will need to make the pull secret available across
     the cluster. This can be achieved by applying the Red Hat Marketplace Pull Secret
     as a [global pull secret](https://docs.openshift.com/container-platform/latest/openshift_images/managing_images/using-image-pull-secrets.html#images-update-global-pull-secret_using-image-pull-secrets).
-    For alternative approachs, please see the official OpenShift [documentation](https://docs.openshift.com/container-platform/latest/openshift_images/managing_images/using-image-pull-secrets.html).\n\n\n###
+    For alternative approachs, please see the official OpenShift [documentation](https://docs.openshift.com/container-platform/latest/openshift_images/managing_images/using-image-pull-secrets.html).\n\n###
```

Comment:
`For alternative approachs`
approaches*

Reply:
corrected